plugin/build.gradle.kts

```diff
@@ -0,0 +1,49 @@
+dependencies {
+    implementation(project(":refactoringminer"))
+}
+
+open class IOCliTask : org.jetbrains.intellij.tasks.RunIdeTask() {
+    // Name of operation to perform (h/help for usage)
+    val operation: String? by project
+
+    // Path to the local git directory or URL
+    val pathToGit: String? by project
```

Comment:
why do we need it? or you mean a path to the git repository?

Reply:
Yes. Not good naming?

Reply:
i thought it is a path to the user's git root. what about `"projectPath"`?

Reply:
Ok

Reply:
Gradle throw exception on this name. I think it is reserved. Suggest `gitProjectPath`

Reply:
looks good to me